controllers/app.controllers.js

```diff
@@ -38,9 +39,9 @@ exports.getReviewById = (req, res, next) => {
 
 exports.getCommentsByReviewId = (req, res, next) => {
   const reviewId = req.params.review_id;
-  selectComments(reviewId)
+  Promise.all([checkReviewIdExists(reviewId), selectComments(reviewId)])
```

Comment:
Great way to differentiate between invalid article and valid article but with no comments 👍